packages/big-design/src/components/Toggle/styled.tsx

```diff
@@ -0,0 +1,44 @@
+import { theme as defaultTheme } from '@bigcommerce/big-design-theme';
+import { hideVisually } from 'polished';
+import styled from 'styled-components';
+
+import { withTransition } from '../../mixins/transitions';
+
+import { ToggleLabelProps } from './Toggle';
+
+export const HiddenCheckbox = styled.input`
+  ${hideVisually()}
+`;
+
+export const StyledToggleLabel = styled.label<ToggleLabelProps>`
+  ${withTransition(['background, border-color'])}
+  background: ${({ checked, theme }) => (checked ? theme.colors.primary30 : theme.colors.secondary30)};
+  border-color: ${({ checked, theme }) => (checked ? theme.colors.primary30 : theme.colors.secondary30)};
+  border-radius: ${({ theme }) => theme.helpers.remCalc(8)};
+  cursor: pointer;
+  height: ${({ theme }) => theme.helpers.remCalc(14)};
+  position: relative;
+  width: ${({ theme }) => theme.helpers.remCalc(38)};
+
+  &::before {
+    ${withTransition(['background, transform'])}
+    top: -${({ theme }) => theme.helpers.remCalc(5)};
+    background: ${({ checked, theme }) => (checked ? theme.colors.primary40 : theme.colors.white)};
+    border-radius: ${({ theme }) => theme.borderRadius.circle};
+    content: '';
+    height: ${({ theme }) => theme.spacing.xLarge};
+    position: absolute;
+    width: ${({ theme }) => theme.spacing.xLarge};
+    left: calc(100% - ${({ theme }) => theme.spacing.large});
+    box-shadow: ${({ theme }) => theme.shadow.raised};
```

Comment:
Bit confused by this there is already 
`border-radius: ${({ theme }) => theme.borderRadius.circle};
  box-shadow: ${({ theme }) => theme.shadow.raised};
`

is the intention to change the order of the two
` 
box-shadow: ${({ theme }) => theme.shadow.raised};
border-radius: ${({ theme }) => theme.borderRadius.circle};
`

Reply:
Like I mentioned above, `theme.shadow.raised` outputs 2 CSS properties preconfigured. So you don't need to set it to any CSS property.

Instead of this
```tsx
box-shadow: ${({ theme }) => theme.shadow.raised};
```

Use it like this:
```tsx
${({ theme }) => theme.shadow.raised}
```

It's just more CSS-y to override those properties outputted via  the cascade.